### PR TITLE
feat: contactcard component

### PR DESCRIPTION
**Pull request by Siristol**
**What has been done**
Made a ContactCard component with name, position, email, phone number, and a picture passed as props.  I made it so that you can click both the mail and the phone number. The positions are able to switch between English and Norwegian. 

**To do**
Name, position, email, and phone number are hard-coded in and I am not sure how to do it with the pictures. They could maybe be imported in the same way as the logo or they could be imported from some folder some other place.

### CODE REVIEW

File: components/contact/Card.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      image: string
+      position: string
+      email: string
+      phone: string
+    }
+  }>()
+</script>
+
+<template>
+  <div class="d-flex flex-column">
```
Comment (AasmundN) [inline components/contact/Card.vue:14]: Think you can remove this wrapper div.

File: components/contact/Card.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          <VImg
+            height="100%"
+            class="rounded-lg"
+            :width="150"
+            :src="content.image"
+            cover
+          />
+        </VCol>
+        <VCol>
+          <VCardTitle class="text-h4">
```
Comment (AasmundN) [inline components/contact/Card.vue:27]: Think a bold font weight here would look nicer.

File: components/contact/Card.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            {{ content.name }}
+          </VCardTitle>
+          <VCardText class="text-h6"> <br />{{ content.position }} </VCardText>
+          <VCardText class="text-body-1">
+            <a :href="`mailto:${content.email}`"> {{ content.email }} </a>
```
Comment (AasmundN) [inline components/contact/Card.vue:32]: Think we should change the font color of links. The blue color does not look that great. Have a look at the color theme implemented in #92.

File: components/contact/Card.vue [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      position: string
+      email: string
+      phone: string
+    }
+  }>()
+</script>
+
+<template>
+  <div class="d-flex flex-column">
+    <VCard class="mx-auto" width="435" variant="flat">
```
Comment (AasmundN) [inline components/contact/Card.vue:15]: Should remove auto margin. Margins should be added when the component is used instead.

**Review by AasmundN — COMMENTED** [review]
Looking nice! I think should look into changing our default font color and font, don't think the current font looks very nice.

**Review by ipeglin — DISMISSED** [review]
LGTM!

**Review by ipeglin — APPROVED** [review]
LGTM

**Review by AasmundN — APPROVED** [review]

**Pull request merged**